web-ui/src/app/components/step/results/results.component.html

```diff
@@ -81,13 +81,80 @@ <h2 class="subtitle">
+                    </div>
+                </div>
+            </div>
+        </h2>
+        <table class="table is-fullwidth">
+            <thead>
+                <tr>
+                    <th>Name</th>
+                    <th>Hits</th>
+                    <th>All Sentences</th>
+                </tr>
+            </thead>
+            <tbody>
+                <ng-container *ngFor="let provider of resultSummary | keyvalue">
+                    <ng-container *ngFor="let corpus of provider.value | keyvalue">
+                        <tr>
+                            <th>
+                                <label class="checkbox">
+                                    <input type="checkbox" [checked]="corpus.value.show" (change)="toggleCorpus(provider.key, corpus.key)" /> {{corpus.key}}
+                                </label>
+                            </th>
+                            <th>{{corpus.value.hits.length}}</th>
+                            <th>{{corpus.value.totalSentences}}</th>
+                        </tr>
+                        <ng-container *ngIf="corpus.value.show">
+                            <tr *ngFor="let component of corpus.value.components | keyvalue">
+                                <td>
+                                    <label class="checkbox">
+                                        <input type="checkbox" [checked]="component.value.show" (change)="toggleComponent(provider.key, corpus.key, component.key)" /> {{component.key}}
+                                    </label>
+                                </td>
+                                <td>{{component.value.hits}}</td>
+                                <td>{{component.value.sentences}}</td>
+                            </tr>
+                        </ng-container>
+                        <tr></tr> <!-- spacer row -->
+                    </ng-container>
+                </ng-container>
+            </tbody>
+            <tfoot>
+                <tr>
+                    <td>
+                        <!--
+
+                            <label class="checkbox">
+                                <input type="checkbox" [checked]="totalSelected" (change)="toggleAllComponents(!totalSelected)" /> Total
+                            </label>
+                        -->
+                    </td>
+                    <td>{{totalHits}}</td>
+                    <td>{{totalSentences.toString()}}</td>
+                </tr>
+            </tfoot>
+        </table>
     </div>
 </div>
 <div class="level">
     <div class="level-left">
-        <h1 class="title" i18n>
-            {filteredResults.length, plural, =1 {1 Result} other {{{filteredResults.length}} Results}} for <i>&ldquo;{{corpus}}&rdquo;</i>
+        <h1 class="title" [ngClass]="{'is-loading': loading}">
+            Results: {{filteredResults.length}}<ng-container *ngIf="(totalHits - filteredResults.length) > 0"> ({{totalHits - filteredResults.length}} hidden)</ng-container>
```

Comment:
Ik had hier eerder de feedback voor gekregen om te laten zien welke corpus geselecteerd is, dus dat is wel handig om te blijven laten zien.

Reply:
De lijst namen staat ook in de counts tabel, is dat niet afdoende? Anders moet ik nog even nadenken hoe het moet werken wanneer er in meerdere treebanks tegelijk wordt gezocht, een lijst van meerdere namen (lassy, cgn, ..., ...) lijkt me ook nog niet ideaal.